scripts/fib.py

```diff
@@ -0,0 +1,10 @@
+def fib(n: int) -> int:
```

Comment:
Add a docstring to the function

```suggestion
def fib(n: int) -> int:
    """Calculate the Fibonacci number for a given integer n."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)